metrics/network/lib/network-test-common.bash

```diff
@@ -36,7 +36,7 @@ function start_server {
 	local server_name="$1"
 	local image="$2"
 	local server_command="$3"
-	$DOCKER_EXE run -d --runtime="${RUNTIME}" --name=${server_name} ${image} \
+	$DOCKER_EXE run -d --runtime="${RUNTIME}" --privileged --name=${server_name} ${image} \
```

Comment:
A question for @mcastelino (and anybody else who knows) - so, is `--privileged` benign in the `cc-runtime` case?
We should correlate that with the limitations doc:
https://github.com/clearcontainers/runtime/blob/master/docs/limitations.md#docker-run---privileged
that currently says 'unsupported'. We can change that to 'benign' or maybe 'impotent' if necessary.